storage/rule_disable.go

```diff
@@ -0,0 +1,227 @@
+// Copyright 2021 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"time"
+
+	"github.com/rs/zerolog/log"
+
+	utypes "github.com/RedHatInsights/insights-operator-utils/types"
+	"github.com/RedHatInsights/insights-results-aggregator/types"
+)
+
+// DisableRuleSystemWide disables the selected rule for all clusters visible to
+// given user
+func (storage DBStorage) DisableRuleSystemWide(
+	orgID types.OrgID, userID types.UserID,
+	ruleID types.RuleID, errorKey types.ErrorKey,
+	justification string) error {
+
+	now := time.Now()
```

Comment:
is this saved/returned in the expected format like other timestamps? I don't remember the exact reason, but the [sql.NullTime](https://github.com/RedHatInsights/insights-results-aggregator/blob/36781f0e7c6b4ba2fa51fb2e17fc13b08e93f4e6/storage/rule_toggle.go#L61) was used for the toggling, but I guess it was only done so because the columns could've been null, which is not possible here

Reply:
yup it is ok there